lib/components/ButtonRenderer/ButtonRenderer.tsx

```diff
@@ -206,10 +248,48 @@ export interface PrivateButtonRendererProps {
   ) => ReactNode;
 }
 
+type PropMapper = {
```

Comment:
This name threw me because this type isn't actually mapping anything. Thoughts on something like `VisualProps` or `ResolveToneAndVariantOptions`, or even just inlining it so we don't have to name it?

Reply:
Yeah good find, i never went back to name it. I'll just inline it.